lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -218,9 +227,20 @@ def _print_filter_warning():
     """)
 
 
+def _print_devices_info(vgs):
+    print("""\
+Configuring LVM to use devices file instead of LVM filter.
```

Comment:
The text "instead of LVM filter" is relevant to the case when we switch from lvm filter to devices file, no?

I think using this would be enough:

    "Configuring LVM system.devices"